### ISSUE TITLE
Wandb runs combined when using resume

### ISSUE THREAD

**Issue by afang-story** [mlfoundations/open_clip#340]
When making multiple runs with resume as latest, they get combined into the same run on wandb and the name gets changed to the latest one, even though the names are unique.
According to https://docs.wandb.ai/ref/python/init this is because resuming needs a unique id or else it will overwrite the old run.
The solution to this problem would be to add an id to the wandb init call main.py.

**Comment by rom1504** [comment]
The intended behavior is to have a single run.
Would you like an option to have multiple runs ?

**Comment by gabrielilharco** [comment]
It's nice to have support for running multiple runs in parallel imo. Would adding `id=args.name` to https://github.com/mlfoundations/open_clip/blob/main/src/training/main.py#L292 fix this?

**Comment by mitchellnw** [comment]
that would probably do it @gabrielilharco

**Comment by gabrielilharco** [comment]
I'll test this and open a PR if it works

**Comment by gabrielilharco** [comment]
Fixed in https://github.com/mlfoundations/open_clip/pull/344
